snap/snapcraft.yaml

```diff
@@ -0,0 +1,102 @@
+name: edgex-device-rest
+base: core18
+type: app
+version: "replace-me"
+version-script: |
+  echo $(cat VERSION)-$(date +%Y%m%d)+$(git rev-parse --short HEAD)
+license: Apache-2.0
+title: EdgeX REST Device Service
+summary: EdgeX device service for REST protocol
+description: |
+  This device service provides easy way for 3'rd party applications, 
+  such as Point of Sale, CV Analytics, etc., to push data into EdgeX 
+  via the REST protocol.
+
+# delhi: 0, edinburgh: 1, fuji: 2, geneva: 3
+epoch: 3
+
+# TODO: add armhf here when that's supported
+architectures:
+  - build-on: arm64
+  - build-on: amd64
+
+grade: stable
+confinement: strict
+
+apps:
+  device-rest-go:
+    adapter: none
+    command: bin/device-rest-go $CONF_ARG $PROFILE_ARG $REGISTRY_ARG
+    daemon: simple
+    environment:
+      CONF_ARG: "--confdir=$SNAP_DATA/config/device-rest-go"
+      PROFILE_ARG: "--profile=res"
+      REGISTRY_ARG: "--registry=consul://localhost:8500"
+      Device_ProfilesDir: "$SNAP_DATA/config/device-rest-go/res"
```

Comment:
Make`Device_ProfilesDir` all uppercase.
A nit but, trying to make all env var upper case moving forward, which is supported in Geneva release.